NetBarcode/NetBarcode.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
this breaks dotNet Framework compatibility
https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0

Possible to use `netstandard2.0` while also supporting the new `SixLabors` libs?

Reply:
Hi @Bogatinov , Thanks for comment, I will check `netstandard2.0` is supporting the new `SixLabors` libs or not and do a necessary update on it.

Reply:
Hi @tommyliew-machines I checked it out
- `SixLabors.Fonts 1.0.1` works on `netstandard2.0`, but `2.0.0` is `net6.0` dependent
- `SixLabors.ImageSharp.Drawing 1.0.0`  works on `netstandard2.0`, but `2.0.0` is `net6.0` dependent
- `SixLabors.ImageSharp 2.1.8` is the last version that allows `netstandard2.0`

I can open a PR and update the libraries
- SixLabors.Fonts 1.0.1
- SixLabors.ImageSharp.Drawing 1.0.0
- SixLabors.ImageSharp 2.1.8

Then we can do 2 changes
- my PR to offer a small bump in v1.7.1.1, which is still compliant with `netstandard2.0`
- this PR to offer update to v2.0.0 which will break net48 compatibility = if Author is okay with that decision :)